spinn_front_end_common/utility_models/abstract_one_app_one_machine_vertex.py

```diff
@@ -71,14 +71,6 @@ def remember_associated_machine_vertex(self, machine_vertex):
         super(AbstractOneAppOneMachineVertex, self).\
             remember_associated_machine_vertex(machine_vertex)
 
-    @overrides(AbstractGeneratesDataSpecification.generate_data_specification)
```

Comment:
flake8 is saying you need to take this out of the imports; while you're there, I think AbstractHasAssociatedBinary can go too (and you can delete get_binary_file_name(...) from above as well).

Reply:
fixed